mappings/net/minecraft/command/argument/RegistryEntryArgumentType.mapping

```diff
@@ -0,0 +1,43 @@
+CLASS net/minecraft/class_7066 net/minecraft/command/argument/RegistryEntryArgumentType
```

Comment:
Hmm, should we use this or #3025 ?

Reply:
I think #3025's `RegistryPredicateArgumentType` is more accurate because the type parameter is `class_7068` is a predicate rather than a specific registry entry. This predicate can be used to match a registry entry, but does not specify a specific registry entry.